paper-submenu.html

```diff
@@ -131,9 +131,9 @@
        * Expand the submenu content.
        */
       open: function() {
-        if (!this.disabled && !this._active) {
+        if (!this.disabled && !this.opened) {
           this.$.collapse.show();
-          this._active = true;
+          this.opened = true;
           this.__trigger && this.__trigger.classList.add('iron-selected');
```

Comment:
when you say "moving this", do you mean all the side effects?  like the `classList` and `collapse.show()` stuff?

if that's what you mean, sure.